runtime/tutor/en/vim-01-beginner.tutor

```diff
@@ -724,7 +724,7 @@ NOTE: You can also read the output of an external command. For example,
  4. [:r](:r) FILENAME              retrieves disk file FILENAME and puts it
                              below the cursor position.
 
- 5. [:r !dir](:r!)                  reads the output of the dir command and
+ 5. [:r !{unix:(ls),win:(dir)}](:r!){unix:(  ),win:( )}                 reads the output of the {unix:(ls),win:(dir)} command and
```

Comment:
@justinmk ugly trick here, any suggestions?

Reply:
seems fine, but could also do:

```suggestion
 5. [:r !{unix:(ls ),win:(dir)}](:r!)                   reads the output of the {unix:(ls),win:(dir)} command and
```

Reply:
or we could drop `()` in the format, so this could be:
```suggestion
 5. [:r !{unix:ls(:r!)   }{win:(dir)(:r!)  }]                 reads the output of the {unix:(ls),win:(dir)} command and
```

Then the `:%s` needs to be updated.